src/api-collector/main.go

```diff
@@ -0,0 +1,218 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/google/go-github/v61/github"
+	"golang.org/x/oauth2"
+	"gopkg.in/yaml.v2"
+)
+
+const API_DOCS_REPO = "api-hub"
+
+type assetInfo struct {
+	id   int64
+	name string
+}
+
+type OpenAPIInfo struct {
+	Version string `yaml:"version"`
+	Title   string `yaml:"title"`
+}
+type OpenAPISpec struct {
+	OpenAPI string      `yaml:"openapi"`
+	Info    OpenAPIInfo `yaml:"info"`
+}
+
+func main() {
+	owner, token := getArgs()
+	ctx := context.Background()
+	client := getAuthenticatedClient(ctx, token)
+	repos, err := getOrgRepos(ctx, owner, client)
+	if err != nil {
+		log.Fatalf("Error fetching repos: %s", err)
+		os.Exit(1)
```

Comment:
`os.Exit(1)` is already called by `log.Fatalf`

> // Fatalf is equivalent to Printf() followed by a call to os.Exit(1).

Reply:
that's absolutely right, https://github.com/eclipse-tractusx/api-hub/pull/1/commits/56376c89a5badf840d259bc5373e98ecdfc203de